src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
We could also change it to "$key=$value" which is also pretty readable imho ? The curly braces here are a bit redundant i guess.